curp/proto/message.proto

```diff
@@ -48,8 +48,21 @@ message AppendEntriesResponse {
     bool success = 2;
```

Comment:
We can add a field in the response to get the last committed entry index. It's used for the calibration.

Reply:
Do you mean this paragraph in Raft paper?
> If desired, the protocol can be optimized to reduce the number of rejected AppendEntries RPCs. For example, when rejecting an AppendEntries request, the follower can include the term of the conﬂicting entry and the ﬁrst index it stores for that term. With this information, the leader can decrement nextIndex to bypass all of the conﬂicting entries in that term; one AppendEntries RPC will be required for each term with conﬂicting entries, rather than one RPC per entry. In practice, we doubt this optimization is necessary, since failures happen infrequently and it is unlikely that there will be many inconsistent entries.

Reply:
We can't use the commitIndex of the follower because if the logs a follower have can be fewer than committed logs.

Reply:
Yes, the followers' log may be fewer  than committed logs, we should also replicate these missing committed logs too. The field I suggest is used to adjust the calibration starting point faster.